examples/locks.ts

```diff
@@ -0,0 +1,92 @@
+// An example of members locating at and locking elements in a slides
+// application.
+import Ably from 'ably/promises';
+import Spaces from '../dist/cjs/Spaces';
+import { Lock } from '../dist/cjs/Locks';
```

Comment:
I did try that, but got the following error:

```
$ npm run examples:locks

> @ably-labs/spaces@0.0.12 examples:locks
> ts-node ./examples/locks.ts

Error: Cannot find module './Space.js'
Require stack:
- /Users/lewis/src/github.com/ably-labs/spaces/src/Spaces.ts
- /Users/lewis/src/github.com/ably-labs/spaces/examples/locks.ts
```

If you don't mind, let's merge and iterate on the examples separately? I think it should probably import `@ably-labs/spaces` but we'd need everything in a release first.

And yeh this was just for illustrative purposes, it could have just been documentation, but I prefer runnable things.

Reply:
agreed, I've tried this as well and was definitely not as simple as I thought